UnityProject/Assets/Scripts/Construction/ConveyorBeltSwitch.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ConveyorBeltSwitch : MonoBehaviour, ICheckedInteractable<HandApply>
+{
+	public SpriteRenderer spriteRenderer;
+
+	public ConveyorBelt[] conveyorBelts;
+
```

Comment:
You will be able to set up all conveyors in editor, but in game `array `won't work. So you will probably need a `List<ConveyorBelt>`, if you are gonna implement construction and connection to new switches.

Reply:
Yeah thx, ive changed it now.